lyluatex.lua

```diff
@@ -1013,7 +1031,10 @@ function ly.get_option(opt)
 end
 
 
-function ly.is_dim (k, v)
+function ly.is_alias() end
```

Comment:
What is this empty function for?

Reply:
It's to keep the same behaviour than others when declaring it as a package option. It could be an arbitrary variable, providdent that its value isn't "reproducible"; I thought it wasn't harder to set it as a function.